### DISCUSSION TITLE
ScriptReference/Animator.SetTrigger

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#592] [Notes]
# ScriptReference/Animator.SetTrigger

https://docs.unity3d.com/ScriptReference/Animator.SetTrigger.html

**Comment by Flewd** [comment]
You can retrieve the current state of a trigger by using [animator.GetBool](https://docs.unity3d.com/ScriptReference/Animator.GetBool.html)
